packages/playground/apps/starter/components/demo-script.ts

```diff
@@ -0,0 +1,137 @@
+export const demoScript = `import * as THREE from "three";
```

Comment:
What is this used for?

Reply:
demo html
try click `html block` button.
![image](https://github.com/toeverything/blocksuite/assets/17165520/8acc3e80-dc6f-4fe2-9a1a-fe294d3519a3)